src/frontend/hooks/server/invites.ts

```diff
@@ -30,25 +30,17 @@ export function useAcceptInvite(projectId?: string) {
 
   return useMutation({
     mutationFn: async ({inviteId}: {inviteId: string}) => {
-      if (!inviteId) return;
       return mapeoApi.invite.accept({inviteId});
     },
-    onSuccess: () => {
-      // This is a workaround. There is a race condition where the project in not available when the invite is accepted. This is temporary and is currently being worked on.
-      setTimeout(() => {
-        Promise.all([
-          queryClient.invalidateQueries({
-            queryKey: [INVITE_KEY],
-          }),
-          queryClient.invalidateQueries({
-            queryKey: [ALL_PROJECTS_KEY],
-          }),
-        ]).then(() => {
-          if (projectId) {
-            switchActiveProject(projectId);
-          }
-        });
-      }, 5000);
+    onSuccess: projectPublicId => {
```

Comment:
`invite.accept()` resolves with the `projectPublicId`, so no need for the hook to accept that parameter anymore, which makes usage simpler